log.sh

```diff
@@ -100,11 +101,6 @@ alias LSERROR='LSLOG 40'
 alias LSCRITICAL='LSLOG 50'
 alias LSLOGSTACK='LSDEBUG Traceback ; LSCALLSTACK'
 
-LSEXCEPTION () {
-  LSLOG 50 $@; LSDEBUG Traceback ; LSCALLSTACK 1
-  exit 1
-}
-
```

Comment:
Why remove this?

Reply:
this may have actually been there before my intervention

Reply:
..or never added